pkgs/by-name/ms/msecli/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  autoPatchelfHook,
+  buildFHSEnv,
+  fetchurl,
+  lib,
+  stdenv,
+  zlib,
+}:
+
+let
+  pname = "msecli";
+  version = "10.01.012024.00";
+
+  src = fetchurl {
+    url = "https://www.micron.com/content/dam/micron/global/public/products/software/storage-executive-software/msecli/msecli-linux.run";
```

Comment:
This url is not version dependent. If in the future this file is updated it _will_ break the hash.

Reply:
Shall I replace the URL with a link from the Internet Archive's Wayback Machine? I have made a new snapshot at https://web.archive.org/web/20240609190007/https://www.micron.com/content/dam/micron/global/public/products/software/storage-executive-software/msecli/msecli-linux.run and have checked that it builds correctly.

Reply:
if possible, please also communicate with the upstream about making reproducible urls :)